manual/manual_intro.rst

```diff
@@ -82,12 +82,12 @@ How To Cite
 
 .. Instructions and link to paper
 
-If you wish to cite tket in any academic publications, we generally recommend citing our `software overview paper <https://doi.org/10.1088/2058-9565/ab8e92>`_ for most cases.
+If you wish to cite tket in any academic publications, we generally recommend citing our `software overview paper <https://iopscience.iop.org/article/10.1088/2058-9565/ab8e92>`_ for most cases.
 
 If your work is on the topic of specific compilation tasks, it may be more appropriate to cite one of our other papers:
 
-- `"On the qubit routing problem" <https://doi.org/10.4230/LIPIcs.TQC.2019.5>`_ for qubit placement (aka allocation, mapping) and routing (aka swap network insertion, connectivity solving).
-- `"Phase Gadget Synthesis for Shallow Circuits" <https://doi.org/10.4204/EPTCS.318.13>`_ for representing exponentiated Pauli operators in the ZX calculus and their circuit decompositions.
+- `"On the qubit routing problem" <https://drops.dagstuhl.de/entities/document/10.4230/LIPIcs.TQC.2019.5>`_ for qubit placement (aka allocation, mapping) and routing (aka swap network insertion, connectivity solving).
```

Comment:
Do you happen to know if its best practice to use the doi link for papers or not?

Reply:
Based on https://liu.cwp.libguides.com/c.php?g=45846&p=291623 and https://ask.library.uic.edu/faq/345899 I think the intent of DOI links is that while the underlying URL might change, the DOI link will still stay the same. So it's probably best practice.

However, whether we do it or not would depend on whether we're following a particular citation style in our documentation, I think.

Do we have a particular style guide that we (Quantinuum) use for external references in papers that are submitted to arXiv? That's what I'd use as a guide to whether to use DOI links here or not.

(arXiv itself is not helpful in this regard - https://info.arxiv.org/help/faq/references.html#references-in-arxiv-documents implies that it doesn't care about a particular citation format, just as long as it's clear and uniquely specifies what it's pointing at.)

Reply:
Gone with the original doi links for now.

Reply:
EDIT: Gone with DOI links for all